addons/GearSwap/export.lua

```diff
@@ -80,33 +84,13 @@ function export_set(options)
     end
     
     local inv = items.inventory
```

Comment:
This variable can be moved below into the `elseif` clause. It can even be removed and its usage just replaced with `items.inventory`.